posts/superseding-bpmodels/index.qmd

```diff
@@ -0,0 +1,51 @@
+---
+title: "Key considerations for retiring/superseding an R package"
+subtitle: "A reflection on the example of epichains and bpmodels"
+author:
+  - name: "James Mba Azam"
+    orcid: "0000-0001-5782-7330"
+  - name: "Sebastian Funk"
+    orcid: "0000-0002-2842-3406"
+date: "2025-01-24"
+categories: [software lifecycles, R, R package, software design, DOI]
+format:
+  html:
+    toc: true
+execute: 
+  cache: true
+---
+
+```{r, echo = FALSE}
+knitr::opts_chunk$set(
+  cache = TRUE,
+  message = FALSE
+)
+```
```

Comment:
Could you please remove this entirely and delete the contents of `index_cache/` since it doesn't seem that you're making use of code in the rest of the post?

Reply:
Resolved in 56239a32ccb65c81afe9fedf66a4e2a8822ca68d